content/en/docs/Installation/gce.md

```diff
@@ -0,0 +1,59 @@
+---
+title: "Installation on Google Cloud"
+linkTitle: "Installation on Google Cloud"
+weight: 3
+date: 2025-02-28
+description: Install Kairos on Google Cloud
+---
+
+This page describes how to install Kairos on Google Cloud after you have created a disk image. Since release v3.3.1, Kairos pipeline is pushing a public OS image to Google Cloud which you can use.
+If you want to build a custom image, you can follow the instructions in the [Build Kairos appliances]({{< relref "../advanced/build" >}}) page.
+
+## Prerequisites
+
+- A Google Cloud account with permissions to create VMs.
+- A Google Cloud compatible image of Kairos. You can use the public image provided by Kairos (see below) or [build your own image]({{< ref "/docs/Reference/auroraboot.md#generate-raw-disk-images" >}}) and upload it to your google project ([check how the Kairos CI does it](https://github.com/kairos-io/kairos/blob/48d5c2bc8fc5555263f799db8a3388d7d46cd559/.github/workflows/upload-cloud-images.yaml#L36-L89)).
+
+## Deploy a VM
+
+Unfortunately Google Cloud [doesn't allow users to search among public images in different projects](https://cloud.google.com/compute/docs/images/managing-access-custom-images#share-images-publicly). One has to know the exact project and name of the image they intend to use, even if its publicly accessible to [all authenticated users](https://cloud.google.com/compute/docs/images/managing-access-custom-images#limitations). Using the public image seems to only be possible through the command line because of the above.
+
+1. Make sure you are authenticated with the cli: `gcloud auth login`
+1. Create a VM using the latest Kairos image:
+
+
+```
```

Comment:
If this only works with Ubuntu 24.04 then you can add the only-flavors here ... if it's with multiple flavors then I guess there should also be a shortcode for that?

```suggestion
```bash {class="only-flavors=Ubuntu+24.04"}
```

Reply:
We only push ubuntu 24.04, hardcoded: https://github.com/kairos-io/kairos/blob/48d5c2bc8fc5555263f799db8a3388d7d46cd559/.github/workflows/upload-cloud-images.yaml#L65

It doesn't matter because people are supposed to reset to whatever image they want.

Reply:
I commited the suggestion but I'm not sure it's right. We don't want to appear as if only ubuntu  is supported on public cloud, because that's not true. It's just that we only push one image because that can be used to install any image. The choice of flavor we push was random.

Reply:
done